app/mailers/tt_application_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class TTApplicationMailer < Mail::Notify::Mailer
+  GENERIC_NOTIFY_TEMPLATE = 'b357c7b4-7e7d-4f59-a97d-301758a13eb6'.freeze
+
+  def send_application(to:, applicant:)
```

Comment:
I don't think I'm comfortable with defining the interface of this as taking an `applicant:` at this stage. Because the actual requirements for emailing applications aren't well understood, it feels over-specific.

Reply:
Sure that sounds reasonable. Do we have an idea what I we can pass into the send_application for now?